### ISSUE TITLE
document BUILD_COQ

### ISSUE THREAD

**Issue by DanGrayson** [UniMath/UniMath#771]
(from the Makefile), so users can bypass the compilation of coq

**Comment by siddharthist** [comment]
There is some documentation [in the Makefile template](https://github.com/UniMath/UniMath/blob/614ef5858bc5620ee289c0bef8d3c6033bc8cdd2/build/Makefile-configuration-template#L17). Are you thinking this should be listed in `INSTALL.md`? I agree that this should be more visible. It should probably mention that Unimath is Coq 8.6+ compatible.

**Comment by DanGrayson** [comment]
Yes, thanks, I'll put it in INSTALL.md and mention the compatibility.
